src/raftstore/store/snap.rs

```diff
@@ -704,14 +706,23 @@ impl Snap {
             } else {
                 let mut key_count = 0;
                 let mut size = 0;
+                let base: i64 = 128 * 1024;
+                let mut left: i64 = 0;
                 snap.scan_cf(
                     cf,
                     &begin_key,
                     &end_key,
                     false,
                     &mut |key, value| {
+                        let l: i64 = key.len() + value.len();
+                        if left < l {
+                            let req = ((l + base - 1) % base) * base;
```

Comment:
Too complicated. 
let step: usize = 128 * 1024;
let mut bytes: usize = 0;
...
if bytes >= base {
  bytes = 0;
  limiter.request(base, 0);
}
bytes += key.len() + value.len();

Reply:
What if key.len() + value.len() > step?

Reply:
It doesn't matter, make it simple is more important.